Google.Api.Generator.Tests/ProtoTests/Paginated/Testing.Paginated/PaginatedClient.g.cs

```diff
@@ -209,8 +209,8 @@ public abstract class PaginatedClient
         /// </summary>
         /// <param name="request">The request object containing all of the parameters for the API call.</param>
         /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
-        /// <returns>A pageable sequence of <see cref="ResourceName"/> resources.</returns>
-        public virtual gax::PagedEnumerable<ResourceResponse, ResourceName> ResourcedMethod(ResourceRequest request, gaxgrpc::CallSettings callSettings = null) =>
+        /// <returns>A pageable sequence of <see cref="string"/> resources.</returns>
```

Comment:
It seems odd to see a string referred to as a resource, but it's probably not worth special-casing it.

Reply:
ack